internal/storage/sql/evaluation_test.go

```diff
@@ -639,16 +639,6 @@ func (s *DBTestSuite) TestGetEvaluationRollouts() {
 
 	require.NoError(t, err)
 
-	_, err = s.store.CreateConstraint(context.TODO(), &flipt.CreateConstraintRequest{
-		SegmentKey: segment.Key,
-		Type:       flipt.ComparisonType_STRING_COMPARISON_TYPE,
-		Property:   "foo",
-		Operator:   "EQ",
-		Value:      "bar",
-	})
-
-	require.NoError(t, err)
```

Comment:
Removed the `CreateConstraint` here to ensure that the query still picks up the necessary information.